plugin-api/src/main/java/org/hyperledger/besu/plugin/services/metrics/PoAMetricsService.java

```diff
@@ -21,7 +21,10 @@
 
 /**
  * Provides relevant data for producing metrics on the status of a Proof of Authority (PoA) node.
+ *
+ * @deprecated Classes which inherit from this interface should move to using PoaQueryService.
```

Comment:
nit: It's not just classes which inherit that need to migrate, also users of the interface.
```suggestion
 * @deprecated This interface has been replaced by {@link PoaQueryService}.
```

Reply:
reworded as below